Makefile

```diff
@@ -97,6 +97,11 @@ all: manager
 .PHONY: ci
 ci: test
 
+# helper to get your gomod up-to-date
+.PHONY: gomod
+gomod:
+	go mod tidy && go mod vendor && (cd cmd/otel-allocator && go mod tidy) && (cd cmd/operator-opamp-bridge && go mod tidy)
```

Comment:
i was getting very tired of gomodding and cding. Happy to remove if this isn't wanted!

Reply:
Well, if we could merge https://github.com/open-telemetry/opentelemetry-operator/pull/2319, then we could use a single module - then no more cding. :sunglasses: